### PR TITLE
Documentation Cleanup: landing page, getting-started, and guide/bundles

### PR DESCRIPTION
**Pull request by JKomoroski**
Here is my first foray into tightening up the documentation. This PR is only over three files, If it is welcomed I will do more.

I attempted to keep the content and meaning the same, only making adjustments to grammar, spelling, and phrasing (idiomatic English is a nightmare, I can't imagine trying to learn the nuances as a second language). 

Now that I've started going through it in detail, I have a list of changes I'd like to make to standardize documentation, but I'll open an issue with those suggestions before I implement them.

### CODE REVIEW

**Review by xvik — COMMENTED** [review]
Thank you very much! I have just a few remarks, could you please correct them (or tell me if I'm wrong in some of them)

File: src/doc/docs/getting-started.md [hunk 1]
```diff
@@ -322,17 +323,18 @@ bootstrap.addBundle(GuiceBundle.builder()
                 .build());
 ```
 
-The only difference is the absence of classpath scan (but you'll have to manually declare all extensions).
+The only difference is the absence of `.enableAutoConfig(...)` and the explicit declaration of desired all extensions.
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/getting-started.md:326]: :+1:

File: src/doc/docs/getting-started.md [hunk 2]
```diff
@@ -322,17 +323,18 @@ bootstrap.addBundle(GuiceBundle.builder()
...
 
 !!! tip
-    Explicit extensions declaration could be used together with classpath scan: for example,
-    classpath scan could not cover all packages with extensions (e.g. due to too much classes)
-    and not covered extensions may be specified manually.    
+    Explicit extension declaration could be used together with `enableAutoConfig` (classpath scan). For example,
+    a classpath scan may only scan for extensions in your application's package and subpackages, while extensions from outside
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/getting-started.md:330]: :+1:

File: src/doc/docs/getting-started.md [hunk 3]
```diff
@@ -349,20 +351,20 @@ public class SampleModule extends AbstractModule {
...
 ```     
 
-Guicey will recognize all (3) bindings and register extensions. The difference with classpath scan 
+Guicey will recognize all three bindings and register extensions. The difference with classpath scanning
 or manual declaration is only that guicey will not declare default bindings for extensions 
 (by default, guicey creates untargetted bindings for all extensions: `bind(Extension.class)`).
 
 !!! tip
-    One extension may be found by classpath scan, declared manually and in binding,
+    One extension may be found by classpath scan and declared manually in binding,
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/getting-started.md:364]: Addressed this.

File: src/doc/docs/guide/bundles.md [hunk 4]
```diff
@@ -103,11 +102,11 @@ For example:
 Bundles are often intended to be used multiple times (for example, [spa bundle](../extras/spa.md)).
 
 But in some cases, only one bundle instance must be installed. For example, [eventbus bundle](../extras/eventbus.md)
-must be installed just once. Or it may be some common bundle, installed by multiple other bundles.
+must be installed just once. The opposite may be true, a common bundle could be installed by multiple other bundles.
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/guide/bundles.md:105]: How embarrassing. Caught by a comma splice. I will be more careful in future PRs.

File: src/doc/docs/guide/bundles.md [hunk 5]
```diff
@@ -103,11 +102,11 @@ For example:
...
 
-In order to solve such cases guicey provides [de-duplication mechanism](deduplication.md).
+In order to manage these such cases guicey provides a [de-duplication mechanism](deduplication.md).
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/guide/bundles.md:107]: :+1:

File: src/doc/docs/getting-started.md [hunk 6]
```diff
@@ -302,13 +303,13 @@ Multiple modules could be registered at once:
 ```
 
 !!! note
-    Registration above occur in dropwizard initialization phase, when neither `Configuration`
-    nor `Environment` objects are available, but if you need them in module then either
+    The above registration occurs in dropwizard initialization phase, when neither `Configuration`
+    nor `Environment` objects are available. If you need access to them in a module then either
```
[the remark this answers is not in the document]
Comment (JKomoroski) [reply src/doc/docs/getting-started.md:307]: In this case I wasn't using the infinitive form of `to access`. I have rephrased to keep it clear.

**Comment by JKomoroski** [comment]
Addressed each of your comments in a second commit.

**Comment by xvik** [comment]
All changes were finally published with an [updated documentation site](http://xvik.github.io/dropwizard-guicey/latest/) (updated features like version switcher, dark theme, new latest version alias - documentation itself remained the same)

**Pull request merged**